app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+# create_posts table
+class CreatePost < ApplicationRecord
+  has_many :comments, foreign_key: :create_posts_id
+  has_many :likes, foreign_key: :create_posts_id
+  belongs_to :author, class_name: 'CreateUser', foreign_key: :author_id

```

Comment:
- [ ] Nice job renaming all your models to be singular as suggested by the previous reviewer, but you forgot to change the class name to `user` for the `belong_to` relation in line 5, Please fix that.

- [ ] Same here, in the `has_many` relations the foreign key suppose to be `post_id`, Kindly change it or remove it entirely, and the naming conventions will create it as `post_id` automatically.

[Hint] It will be like that:
```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```

Reply:
fixed